kpops/components/streams_bootstrap/__init__.py

```diff
@@ -1,26 +1,61 @@
+from __future__ import annotations
+
+import logging
 from abc import ABC
+from typing import TYPE_CHECKING
 
+import pydantic
 from pydantic import Field
 
 from kpops.component_handlers.helm_wrapper.model import HelmRepoConfig
-from kpops.components.base_components.helm_app import HelmApp
+from kpops.component_handlers.kubernetes.utils import (
+    IMAGE_TAG_PATTERN,
+)
+from kpops.components.base_components.helm_app import HelmApp, HelmAppValues
 from kpops.utils.docstring import describe_attr
 
+if TYPE_CHECKING:
+    try:
+        from typing import Self  # pyright: ignore[reportAttributeAccessIssue]
+    except ImportError:
+        from typing_extensions import Self
+
 STREAMS_BOOTSTRAP_HELM_REPO = HelmRepoConfig(
     repository_name="bakdata-streams-bootstrap",
     url="https://bakdata.github.io/streams-bootstrap/",
 )
 STREAMS_BOOTSTRAP_VERSION = "2.9.0"
 
+log = logging.getLogger("StreamsBootstrap")
+
+
+class StreamsBootstrapValues(HelmAppValues):
+    """Base value class for all streams bootstrap related components.
+
+    :param image_tag: Docker image tag of the streams-bootstrap app.
+    """
+
+    image_tag: str = Field(
+        default="latest",
+        pattern=IMAGE_TAG_PATTERN,
+        description=describe_attr("image_tag", __doc__),
+    )
+
 
 class StreamsBootstrap(HelmApp, ABC):
     """Base for components with a streams-bootstrap Helm chart.
 
+    :param app: Streams bootstrap app values
```

Comment:
nit but let's try to keep the naming consistent
```suggestion
    :param app: streams-bootstrap app values
```